pkg/kv/dynamodb/store.go

```diff
@@ -322,6 +322,11 @@ func (s *Store) setWithOptionalPredicate(ctx context.Context, partitionKey, key,
 		if usePredicate && errors.As(err, &errConditionalCheckFailed) {
 			return kv.ErrPredicateFailed
 		}
+		var errRequestLimitExceeded *types.RequestLimitExceeded
+		var errProvisionedThroughputExceededException *types.ProvisionedThroughputExceededException
+		if errors.As(err, &errRequestLimitExceeded) || errors.As(err, &errProvisionedThroughputExceededException) {
+			return kv.ErrSlowDown
```

Comment:
This propagates to openapi controller too. Do we change the returned status code or anything else in the response as a result?

Reply:
No, it returns an internal server error

Reply:
@Jonathan-Rosenberg at the time Barak added http error 420 to all endpoints. We need to handle this in the controller and set the right code for this error.